### PR TITLE
fix: text selection occurs when selecting cells

### PR DESCRIPTION
**Pull request by seonim-ryu**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description

Resolves the problem of text selection when selecting cells.

**AS-IS**

cell-selection-wrong

**TO-BE**

cell-selection-correct

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료
고생하셨습니다!

File: apps/editor/src/wysiwyg/plugins/tableSelection/tableSelectionView.ts [hunk 1]
```diff
@@ -51,6 +51,9 @@ export default class TableSelection {
   }
 
   handleMousemove(ev: Event) {
+    // @TODO this should be processed to block the event only when another cell comes out
+    ev.preventDefault();
```
Comment (js87zz) [inline apps/editor/src/wysiwyg/plugins/tableSelection/tableSelectionView.ts:55]: 이 부분을 호출안해줘서 텍스트 셀렉션이 생겼었나 보네요ㅎㅎ

**Review by js87zz — APPROVED** [review]
고생많으셨습니다!

File: apps/editor/src/wysiwyg/plugins/tableSelection/tableSelectionView.ts (hunk 1 above)
Comment (seonim-ryu) [reply apps/editor/src/wysiwyg/plugins/tableSelection/tableSelectionView.ts:55]: 이것보다는.. `CellSelection` 쪽에서 `$anchor`, `$head` 값이 오버라이드 되면서 셀렉션이 발생하는거였어요ㅎ
`$anchor` -> `startCell`, `$head` -> `endCell` 이렇게 셀 노드를 따로 관리해야지 셀렉션이 발생하지 않아서.. 그 부분이 핵심입니다ㅎㅎ

**Pull request merged**